### PR TITLE
Upgrade golang to 1.19.4, other dependencies as needed

### PR DESCRIPTION
**Pull request by briantopping**
Signed-off-by: Brian Topping <brian.topping@sap.com>

**How to categorize this PR?**

/area security
/kind enhancement
/platform vsphere

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
Upgrade golang to 1.19.4, other dependencies as needed
```

### CODE REVIEW

**Comment by ThormaehlenFred** [comment]
@briantopping I am not able to approve. This should be done by the reviewers.

File: cmd/gardener-extension-provider-vsphere/main.go [hunk 1]
```diff
@@ -19,16 +19,16 @@ package main
...
 
 	"github.com/gardener/gardener-extension-provider-vsphere/cmd/gardener-extension-provider-vsphere/app"
 
-	"github.com/gardener/gardener/pkg/logger"
 	runtimelog "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 func main() {
-	runtimelog.SetLogger(logger.ZapLogger(false))
+	runtimelog.SetLogger(utils.ZapLogger(false))
```
Comment (MartinWeindel) [inline cmd/gardener-extension-provider-vsphere/main.go:31]: Please use something like
```
runtimelog.SetLogger(logger.MustNewZapLogger(logger.InfoLevel, logger.FormatJSON))
```
instead.

**Review by MartinWeindel — CHANGES_REQUESTED** [review]
Looks good overall. I have only one minor change request to keep the logging similar to other extensions.

**Review by MartinWeindel — APPROVED** [review]
/lgtm

**Pull request merged**